style.css

```diff
@@ -0,0 +1,417 @@
+  height: 200px;
+  font-weight: 700;
+  font-size: 40px;
+  padding: 90px 12px 170px;
+  line-height: 52px;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.container .me h1 {
+  font-size: 40px;
+}
+
+.container .bio {
+  width: 327px;
+  height: 146px;
+  left: 0;
+  top: 0;
+  padding: 85px 12px 24px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.container .lets_contact {
+  width: 147px;
+  height: 24px;
+  margin-top: 90px;
+  padding-left: 12px;
+}
+
+.container .lets_contact .contact {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 11px;
+  line-height: 24px;
+  color: #7f8cff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.social_media {
+  padding-top: 0%;
+  margin: 0 auto;
+  display: flex;
+}
+
+.social-icons-set {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.social-icon {
+  list-style: none;
+  margin: 10px;
+}
+
+.social-icon a:hover {
+  color: #6070ff;
+}
+
+.social-icon a i {
+  font-size: 20px;
+}
+
+.social-icon + .social-icon {
+  margin-left: 10px;
+}
+
+/* coming soon */
+
+/* Start middl section  */
+
+.middle_section {
+  height: 552px;
+  width: 343px;
+  margin: 40px 16px 0 16px;
+}
+
+.middle_section div.pic {
+  width: 295px;
+  height: 220px;
+  margin: 16px 20px;
+  background-image: url(Snapshoot\ Portfolio.png);
+  background-size: cover;
+}
+
+.middle_section div.name {
+  width: 295px;
+  height: 40px;
+  margin: 5px 20px;
+}
+
+.middle_section div.descript {
+  width: 295px;
+  height: 30px;
+  margin: 5px 20px;
+  display: flex;
+}
+
+.middle_section div.descript div.camp {
+  width: 30%;
+  color: #344563;
+}
+
+.middle_section div.descript div.year {
+  width: 70%;
+  color: #7a869a;
+  font-size: 13px;
+}
+
+.middle_section div.descript div.year ul.list {
+  display: flex;
+  justify-content: start;
+}
+
+.middle_section div.descript div.year ul.list li {
+  margin: 0 15px;
+}
+
+.middle_section div.task {
+  width: 295px;
+  height: 76px;
+  margin: 5px 20px;
+}
+
+.middle_section div.task p {
+  color: #344563;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.middle_section div.web {
+  width: 295px;
+  height: 24px;
+  margin: 0 20px;
+  display: flex;
+}
+
+.middle_section div.web div.right {
+  width: 80%;
+  display: flex;
+  justify-content: center;
+}
+
+.middle_section div.web div.right ul {
+  list-style-type: none;
+  display: flex;
+  justify-content: space-around;
+  width: 100%;
+}
+
+.middle_section div.link a {
+  text-decoration: none;
+  color: #396df2;
+  font-size: 17px;
+  font-weight: 500;
+  padding: 5px 0 0 4px;
+  height: 100%;
+  width: 100%;
+}
+
+section.about div.link a {
+  text-decoration: none;
+  color: #396df2;
+  font-size: 17px;
+  font-weight: 500;
+  padding: 7px 0 0 13px;
+  height: 100%;
+  width: 100%;
+}
+
+.middle_section div.link a:hover {
```

Comment:
- [ ] great job with the button interactions. kindly use the correct hover color of `#6070FF` like in the original design. please add this for all the buttons